web/json_options.go

```diff
@@ -20,20 +20,51 @@ import "time"
 // JSONOption values passed to ConvertJSONObjectList.
 type jsonOptions struct {
 	complexAttributeNameDelimiter string
-	dateTimeFormats               []string
+	dateTimeFormats               []DateTimeFormatWithTimeZone
+
+	// localTimeZoneOffset is the default local timezone offset that should
+	// be used for parsing date-time attributes lacking any time zone info.
+	// This should be set to the number of seconds east of UTC. If this is
+	// set to 0, this will default to UTC.
```

Comment:
Very minor nit. You can ignore.
```suggestion
	// localTimeZoneOffset is the default local timezone offset, as a number of seconds east of UTC,
	// to be used for parsing date-time attributes lacking any time zone info.
	// Defaults to 0, i.e. the UTC timezone.
```

Reply:
I like your comment better - I've went ahead and updated this